Makefile.dep

```diff
@@ -224,3 +224,7 @@ endif
 ifneq (,$(filter ng_at86rf2%,$(USEMODULE)))
   USEMODULE += ng_at86rf2xx
 endif
+
+ifneq (,$(filter kw2xrf,$(USEMODULE)))
+  USEMODULE += kw2xrf
```

Comment:
I mean that this creates a dependency from A to A. The Atmel driver maps from a wild card (2%%) to 2xx.